app/src/main/java/com/example/sharingang/NewItemFragment.kt

```diff
@@ -124,5 +163,8 @@ class NewItemFragment : Fragment() {
     private fun updateLocation(location: Location) {
         binding.latitude = location.latitude.toString()
         binding.longitude = location.longitude.toString()
+        val address =
+            geocoder.getFromLocation(location.latitude, location.longitude, 1).getOrNull(0)
```

Comment:
This logic is repeated above, might be worth extracting into a method?

Reply:
Where is the logic repeated? If you're referring to `geocoder.getFromLocationName(place.address, 1).getOrNull(0)`, it's not the same: one goes from an address to a location, the other goes from a location to an address. 
Are you referring to the fact that the last argument is 1 and then we have `.getOrNull(0)`? If so, I don't know how to extract that any further, given how different the two methods are.

Reply:
Oh you're right, it's `FromLocationName` instead, woops. Yeah, the methods look somewhat similar but are just opposites :P

Reply:
I understand the confusion, no problem! :D